docs/hidden.md

```diff
@@ -0,0 +1,125 @@
+---
+id: hidden
+title: Hidden
+---
+
+import { ComponentTheme } from '../src/components';
+
+Hidden is used to responsively, according to colorMode or according to platform toggle the visibility value of child components. It doesn't mount the child components in the restricted values of props.
```

Comment:
Making the sentence easier to understand => Hidden is used to toggle the visibility value of child components responsively, based on the colorMode or based on the platform. It doesn't mount the child components in the restricted values of props.